winch/codegen/src/masm.rs

```diff
@@ -229,6 +231,15 @@ impl OperandSize {
             _ => panic!("Invalid bytes {} for OperandSize", bytes),
         }
     }
+
+    /// Convert to I32 and I64.
+    pub fn to_ty(self) -> Type {
+        match self {
+            OperandSize::S32 => types::I32,
+            OperandSize::S64 => types::I64,
+            _ => unimplemented!(),
+        }
+    }
```

Comment:
A conversion [already exists](https://github.com/bytecodealliance/wasmtime/blob/main/winch/codegen/src/isa/aarch64/asm.rs#L22) between Winch's `OperandSize` and Cranelift's aarch64 `OperandSize`; I believe we could get drop this method and instead do something like

```rust
size.into().to_ty()
```

Reply:
done!